src/briefcase/platforms/macOS/utils.py

```diff
@@ -246,6 +246,8 @@ def merge_app_packages(
                             # The file doesn't exist yet; copy it as is, and store the
                             # digest for later comparison
                             self.tools.shutil.copyfile(source_path, target_path)
+                            # Ensure permissions as well.
+                            self.tools.shutil.copymode(source_path, target_path)
```

Comment:
`shutil.copy` looks like a simpler way of doing this.

Reply:
Gotta love an API that has *3* different ways to copy a file... excluding the handful of other options in other modules... 🙄